kafka-mesos.properties

```diff
@@ -5,7 +5,7 @@ user=vagrant
 
 storage=file:kafka-mesos.json
 
-master=zk://master:2181/mesos
+master=master:5050
```

Comment:
I was thinking we should maybe change the default storage to be zk also (and let people know they can use the file json) and keep zk:// for the master value